Fix typo in ApiException of caseInsensitiveMap

samburville opened the pull request:
Not sure if this project is being updated now it's stable, but if it is, then this commit may as well be included in the next update

Commit:
Fix typo in ApiException of caseInsensitiveMap

peter279k commented on MasterCard/Core/Exception/ApiException.php:
```diff
@@ -147,13 +147,13 @@ private function isAssoc($arr)
-            $caseInsesitiveMap = new CaseInsensitiveMap();
-            $caseInsesitiveMap->setAll($errorData);
-            $this->rawErrorData = $caseInsesitiveMap;
+            caseInsensitiveMap = new CaseInsensitiveMap();
```
It misses the `$`.

It should be `$caseInsensitiveMap`.

samburville replied:
Thanks - probably too hastily done by me.
Have fixed to $caseInsensitiveMap now

Commit:
Add missing $ from rename variable in ApiException

This was accidentally removed with last commit when caseInsensitiveMap
was renamed
